<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>371</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_prevpage">
      <property name="text">
       <string>이전 페이지</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_nextpage">
      <property name="text">
       <string>다음 페이지</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_newsImage">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>newsImage</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
